types/nodemark/README.md

```diff
@@ -0,0 +1,14 @@
+# @types/nodemark
```

Comment:
why do you want dedicated license? btw, it's different from native package license, which is MIT

Reply:
Please remove this file.

Reply:
I've no problem to remove the readme and different license.
But I wonder if this repo discourages adding type packages in different license?